tools/devtool

```diff
@@ -852,6 +855,9 @@ cmd_fmt() {
         cargo fmt --all
 }
 
+cmd_mkdocs() {
+    cmd_sh "cargo doc --workspace --no-deps --document-private-items"
```

Comment:
We can get a searchable index page by running
```suggestion
    cmd_sh RUSTDOCFLAGS="--enable-index-page -Zunstable-options" cargo +nightly doc --document-private-items --workspace
```

![image](https://github.com/firecracker-microvm/firecracker/assets/16856679/7f5723b3-3bc9-4d3a-b616-8e405c27e35e)

Reply:
Yeah when that becomes stable we can add it. For now I would rather not add more nightly dependencies.